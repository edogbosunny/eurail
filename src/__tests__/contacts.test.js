
import React from 'react';
import { shallow } from 'enzyme';
import * as redux from 'react-redux'

import Contact from '../component/contacts'


const spy = jest.spyOn(redux, 'useSelector')
spy.mockReturnValue({});
describe('Login page', () => {
  const wrapper = shallow(
    <Contact />,
  );
  it('contacts snap shot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});


