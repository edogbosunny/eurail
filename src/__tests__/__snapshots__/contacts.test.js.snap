// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page contacts snap shot 1`] = `ShallowWrapper {}`;
